craft_parts/executor/environment.py

```diff
@@ -281,10 +281,10 @@ def _replace_attr(
 
     if isinstance(attr, dict):
         result: Dict[str, str] = {}
-        for key, value in attr.items():
+        for _key, _value in attr.items():
```

Comment:
Note to other reviewers: These changes are for [PLW2901](https://docs.astral.sh/ruff/rules/redefined-loop-name/), changing the contents of the loop variable.